.env.prd

```diff
@@ -0,0 +1,12 @@
+# Flask server configuration.
```

Comment:
You've accidentally leaked credentials.

Delete this file (and if you care about your Trello account at all, rotate the key)

If you want to keep a copy locally for whatever reason (though the .env file should suffice), then you can add .env.prd to your .gitignore file and run `git rm --cached .env.prd` in your terminal.

You didn't add it to your .dockerignore either so it's currently getting included in the `COPY` command in your Dockerfile. We do not want any credentials baked into the image.